### ISSUE TITLE
Impossible to edit tags

### ISSUE THREAD

**Issue by LeSpocky** [nextcloud/bookmarks#839]
I can not edit tags anymore with v2.2.0. :disappointed: 

**Describe the bug**
Nothing happens if I click on a link in the overview page. And nothing happens if I click on the "Details" button.

**To Reproduce**
1. Go to your nextcloud website in a desktop web browser
2. Click on the star in the upper bar to get to the bookmarks page
3. Click on the right part of a link row in list view or a thumbnail in the other view or click the three dots and then "Details"
4. Nothing happens

**Expected behavior**
Some dialog should open which would allow me to edit tags.

**Screenshots**
![nextcloud_bookmarks_bug](https://user-images.githubusercontent.com/661636/67181169-14dea100-f3dc-11e9-89c2-bcf6b9cb58f4.png)

**Desktop**
- Debian GNU/Linux 9 (stretch)
  - Firefox 60.9.0esr (64-Bit), 
  - Chromium Version 73.0.3683.75 (Developer Build) built on Debian 9.8, running on Debian 9.11 (64-bit)
- Windows 7 (64 Bit)
  - Firefox 69.0.3 (32-Bit)
  - Google Chrome Version 77.0.3865.120 (64-Bit)

**Server**
 - OS: Debian GNU/Linux 10 (buster)
 - HTTP server: lighttpd/1.4.53 (ssl) - a light and fast webserver
 - Database: mariadb  Ver 15.1 Distrib 10.3.17-MariaDB, for debian-linux-gnu (i686) using readline 5.2
 - PHP version: PHP 7.3.9-1~deb10u1 (cgi-fcgi) (built: Sep 18 2019 10:33:23)
 - Nextcloud version: 16.0.5
 - Bookmarks app version: 2.2.0
- Activated Nextcloud Apps: 
  - Enabled:
    - accessibility: 1.2.0
    - activity: 2.9.1
    - admin_audit: 1.6.0
    - bookmarks: 2.2.0
    - bruteforcesettings: 1.4.0
    - calendar: 1.7.1
    - checksum: 0.4.3
    - cloud_federation_api: 0.2.0
    - comments: 1.6.0
    - contacts: 3.1.6
    - dashboard: 6.0.0
    - dav: 1.9.2
    - federatedfilesharing: 1.6.0
    - federation: 1.6.0
    - files: 1.11.0
    - files_antivirus: 2.2.0
    - files_markdown: 2.1.0
    - files_pdfviewer: 1.5.0
    - files_rightclick: 0.15.1
    - files_sharing: 1.8.0
    - files_texteditor: 2.8.0
    - files_trashbin: 1.6.0
    - files_versions: 1.9.0
    - files_videoplayer: 1.5.0
    - firstrunwizard: 2.5.0
    - gallery: 18.3.0
    - logreader: 2.1.0
    - lookup_server_connector: 1.4.0
    - maps: 0.1.2
    - nextcloud_announcements: 1.5.0
    - notes: 3.0.3
    - notifications: 2.4.1
    - oauth2: 1.4.2
    - password_policy: 1.6.0
    - privacy: 1.0.0
    - provisioning_api: 1.6.0
    - ransomware_detection: 0.5.2
    - ransomware_protection: 1.4.0
    - recommendations: 0.4.0
    - serverinfo: 1.6.0
    - sharebymail: 1.6.0
    - social: 0.2.101
    - spreed: 6.0.4
    - support: 1.0.0
    - survey_client: 1.4.0
    - systemtags: 1.6.0
    - tasks: 0.11.3
    - theming: 1.7.0
    - twofactor_backupcodes: 1.5.0
    - updatenotification: 1.6.0
    - viewer: 1.1.0
    - workflowengine: 1.6.0
  - Disabled:
    - encryption
    - files_external
    - gpxpod
    - ojsxc
    - user_ldap
 - Nextcloud external user backend: none

**Additional context**
The Firefox developer tools showed no network requests when clicking like described above.

**Nextcloud log (nextcloud/data/nextcloud.log)**
No entries when clicking like described above, viewed like this:
```
tail -F /var/lib/owncloud/data/nextcloud.log | jq 'select(.app == "bookmarks")'
```
(However a lot of requests to `"/index.php/apps/bookmarks/bookmark/402/image"` happen again and again with error 404 when just hovering over the thumbnails.)

**Comment by alastair-dm** [comment]
Seems like a duplicate of #814 which was closed prematurely.

**Comment by marcelklehr** [comment]
Can you try with v2.3.0? This should be fixed.

**Comment by LeSpocky** [comment]
I'll try with 2.3.0 once I get the notification in my nextcloud instance. FWIW, here the output from the Firefox devtools console:
```
Error: No language found.
Stack trace:
e/</u<@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:713:15668
e/<@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:713:15371
e@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:713:15321
addedDate@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:947:47836
fn.prototype.get@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:26648
fn.prototype.evaluate@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:27805
yn/<@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:29717
Ve@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:947:46550
t.prototype._render@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:35344
o@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:64429
fn.prototype.get@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:26648
fn.prototype.run@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:27539
dn@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:25650
ee/<@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:12313
Xt@https://cloud.home.lespocky.de/apps/bookmarks/js/bookmarks.main.js?v=2d0798ae-0:7:11755
o@https://cloud.home.lespocky.de/core/js/dist/main.js?v=2d0798ae-0:230:27381
H/<@https://cloud.home.lespocky.de/core/js/dist/main.js?v=2d0798ae-0:230:27526
u@https://cloud.home.lespocky.de/core/js/dist/main.js?v=2d0798ae-0:235:77644
 vue.runtime.esm.js:1888:4
```

**Comment by LeSpocky** [comment]
Seems to be fixed with v2.3.1. :+1:
